.github/workflows/publish.yaml

```diff
@@ -0,0 +1,78 @@
+name: '🏗️ Build and Publish jellyfin-ffmpeg'
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build_publish_debian:
+    uses: jellyfin/jellyfin-ffmpeg/.github/workflows/_meta.yaml@master
+    if: ${{ !github.event.release.prerelease }}
+    with:
+      distro: 'debian'
+      codenames: "[\"buster\", \"bullseye\"]"
+      architectures: "[\"amd64\", \"arm64\", \"armhf\"]"
+      release: true
+
+  build_publish_ubuntu:
+    uses: jellyfin/jellyfin-ffmpeg/.github/workflows/_meta.yaml@master
+    if: ${{ !github.event.release.prerelease }}
+    with:
+      distro: 'ubuntu'
+      codenames: "[\"focal\", \"bionic\", \"impish\", \"hirsute\"]"
+      architectures: "[\"amd64\", \"arm64\", \"armhf\"]"
+      release: true
+
+  build_publish_windows:
+    uses: jellyfin/jellyfin-ffmpeg/.github/workflows/_meta.yaml@master
+    if: ${{ !github.event.release.prerelease }}
+    with:
+      distro: 'windows'
+      codenames: "[\"Windows 10\"]"
+      architectures: "[\"win64\"]"
+      release: true
+
+  maintain_repository:
+    name: Maintain Repository
+    if: ${{ !github.event.release.prerelease && inputs.release }}
+    runs-on: ubuntu-latest
+    needs:
+      - build_publish_debian
+      - build_publish_ubuntu
+      - build_publish_windows
+    strategy:
+      fail-fast: false
+      matrix:
+        distro:
+          - debian
+          - ubuntu
+          - windows
+    steps:
+      - name: Sync mirrors and update symlinks
+        uses: appleboy/ssh-action@v0.1.4
+        env:
+          DISTRO: ${{ matrix.distro }}
+          RELEASE: ${{ github.release.tag_name }}
+        with:
+          host: ${{ secrets.deploy-host }}
+          username: ${{ secrets.deploy-user }}
+          key: ${{ secrets.deploy-key }}
+          script_stop: true
+          envs: DISTRO, RELEASE
```

Comment:
this certainly works but you could also simply use the GH expression in the script section as it will get evaluated and interpolated before it gets send of to be run. (that is also applicable for the regular `run` blocks)

Reply:
I just adapted what you did here: https://github.com/jellyfin/jellyfin-meta-plugins/blob/master/.github/workflows/publish.yaml

Reply:
yeah, I did that there to have the 2 variables out of the script and within the Workflow. I however also have expression syntax in there.
Anyway, this is just a comment as to what is possible, use what you find convenient and passing the envs over might be the easier to write/read way anyway.

Reply:
I changed it nonetheless. We can change it again if it does not work